Documentation/gettingstarted/kind.rst

```diff
@@ -54,13 +54,31 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v2
-   needs to be enabled by setting the kernel ``systemd.unified_cgroup_hierarchy=1`` parameter.
+   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), kind nodes
+   need to have their own cgroup namespaces, and are different from the cgroup namespace
+   of the underlying host so that Cilium can attach BPF programs at the right
+   cgroup hierarchy. To verify this, run the following commands inside
+   respective kind nodes, and ensure that the cgroup values are different.
+
+   .. code-block:: shell-session
+
+      $ docker exec -it kind-control-plane bash
+      $ ls -al /proc/self/ns/cgroup
+      lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532461]'
+
+      $ docker exec -it kind-control-plane bash
+      $ ls -al /proc/self/ns/cgroup
+      lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532543]'
```

Comment:
:thinking: This looks like it's running the same command in the same node, why would the cgroup be different?

Reply:
😄 Copy and paste error - fixed!